qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -273,11 +273,25 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
 
 
 class FineXAmplitude(FineAmplitude):
-    r"""A fine amplitude experiment with all the options set for the :math:`\pi`-rotation."""
+    r"""A fine amplitude experiment with all the options set for the :math:`\pi`-rotation.
+
+    # section: overview
+
+        :class:`FineXAmplitude` is a subclass of :class:`FineAmplitude` and is used to set
+        the appropriate values for the default options.
+    """
 
     @classmethod
     def _default_experiment_options(cls) -> Options:
-        """Default values for the fine amplitude experiment."""
+        r"""Default values for the fine amplitude experiment.
+
+        Experiment Options:
+            add_sx (bool): This option is True by default when calibrating gates with a target

```

Comment:
Is this same with one in `FineAmplitude`? In this case the parser automatically checks parent class. Same for others.

Reply:
The doc strings are tailored to the subclasses.